test/api/unit/services/Mailer.test.js

```diff
@@ -23,16 +23,41 @@ describe('mailer', () => {
     });
 
     describe('.sendUAAInvite()', () => {
-      it('adds a `uaa-invite` job to the mail queue', async () => {
-        const email = 'foo@bar.gov';
-        const link = 'https://foobar.gov';
+      const email = 'foo@bar.gov';
+      const link = 'https://foobar.gov';
+      const orgName = 'test-org';
 
-        const job = await Mailer.sendUAAInvite(email, link);
+      it('adds a `uaa-invite` job to the mail queue for cloud.gov IDP users', async () => {
+        const origin = 'cloud.gov';
+
+        const job = await Mailer.sendUAAInvite(email, link, origin, orgName);
+
+        expect(job.name).to.eq('uaa-invite');
+        expect(job.data.to).to.deep.eq([email]);
+        expect(job.data.html).to.eq(Templates.uaaInvite({ link }));
+      });
+
+      it('adds a `uaa-invite` job to the mail queue for uaa IDP users', async () => {
+        const origin = 'uaa';
+
+        const job = await Mailer.sendUAAInvite(email, link, origin, orgName);
 
         expect(job.name).to.eq('uaa-invite');
         expect(job.data.to).to.deep.eq([email]);
         expect(job.data.html).to.eq(Templates.uaaInvite({ link }));
       });
+
+      it('adds a `uaa-invite` job to the mail queue for agency IDP users', async () => {
```

Comment:
Here you call it agency IDP, but the component name was uaaAgency. Could these match? Either UAA or IDP? (I prefer IDP)

Reply:
Updated